lib/lch.ts

```diff
@@ -0,0 +1,6 @@
+export interface LchValue {
```

Comment:
We already have it here
https://github.com/evilmartians/oklch-picker/blob/0e6505002ec1bdad73375b5800732278b857df16/stores/current.ts#L10-L15

Reply:
not anymore. I moved this declaration in separated file, because generator shouldnt depend on stores

Reply:
In this case you need to define another type. `LchValue` is a specific type of the current value (it is not equal to color type by a reason).

I think generator could use `Lch` from `culori`.